test/unit/src/index.js

```diff
@@ -0,0 +1,238 @@
+import fs from 'graceful-fs';
```

Comment:
You will have to define the constants that were in common to get these tests to work.